Makefile

```diff
@@ -379,7 +379,7 @@ endif
 start-upgrade-test: zetanode-upgrade solana
 	@echo "--> Starting upgrade test"
 	export LOCALNET_MODE=upgrade && \
-	export UPGRADE_HEIGHT=225 && \
+	export UPGRADE_HEIGHT=240 && \
```

Comment:
NIT : `start-upgrade-import-mainnet-test` also uses this height , maybe consider using a constant for both values.